proto/proto/store/ntx_builder.proto

```diff
@@ -98,6 +102,15 @@ message UnconsumedNetworkNotes {
     repeated note.NetworkNote notes = 2;
 }
 
+// GET NETWORK ACCOUNTS
+// ================================================================================================
+
+// Represents the result of getting the network account ids.
+message NetworkAccounts {
+  // The list of network accounts.
+  repeated account.AccountId accounts = 1;
+}
```

Comment:
I wonder if we should name this `NetworkAccountIdList` or something similar. Or at the very least, we should update doc comment on line 110 to say that this is a list of account IDs.